deepchem/lightning/tests/test_dc_lightning_module.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+import deepchem as dc
+from deepchem.models import MultitaskClassifier
+import pytorch_lightning as pl
+import numpy as np
+import torch
+from torch.utils.data import DataLoader
+from deepchem.lightning.dc_lightning_module import DCLightningModule
+
+
+class MolnetDataset(torch.utils.data.Dataset):
+
+  def __init__(self, dataset):
+    self._samples = dataset
+
+  def __len__(self):
+    return len(self._samples)
+
+  def __getitem__(self, index):
+    y = np.zeros((1, 2))
+    y[0, int(self._samples.y[index][0])] = 1.0
+    return (
+        self._samples.X[index],
+        y,
+        self._samples.w[index],
+    )
+
+
+class MolnetDatasetBatch:
+
+  def __init__(self, batch):
+    X = [np.array([b[0] for b in batch])]
+    y = [np.array([b[1] for b in batch])]
+    w = [np.array([b[2] for b in batch])]
+    self.batch_list = [X, y, w]
+
+
+def collate_dataset_wrapper(batch):
+  return MolnetDatasetBatch(batch)
+
+
+class TestDCLightningModule(unittest.TestCase):
+
+  def test_multitask_classifier(self):
+    tasks, datasets, _ = dc.molnet.load_hiv(featurizer='ECFP',
```

Comment:
HIV is a quite large dataset (40K datapoints). Can you replace in the unit test with a small synthetic dataset of ~10 points?